.travis.yml

```diff
@@ -1,21 +1,23 @@
 language: python
 
+sudo: false
+
 python:
   - "2.7"
   - "2.6"
+  - "3.4"
+
+addons:
+  apt:
+    packages:
+      - libgeos-c1
+      - libxml2-dev
 
 install:
-  - sudo apt-get install -y libgeos-c1
   - pip install . --use-mirrors
-  - pip install -r ./requirements-standalone.txt
-  - pip install -r ./requirements-dev.txt
-  - python ./setup.py -q install
-
-before_script:
-  - sudo add-apt-repository ppa:gcpp-kalxas/ppa-tzotsos -y
```

Comment:
There has to be a change in libxml used by Travis, since the libxml version in Precise fails to parse GML. This is why I used my own backport here.